src/components/NotificationContainer.tsx

```diff
@@ -139,19 +149,49 @@ function NotificationsContainer() {
     lastShownNotification.current = latestNotificationItem
   }
 
-  return (
+  const showDialog = React.useCallback(() => {
```

Comment:
Two things, also concerns `closeDialog()`:
- Rename the variable: When reading the code I am not so much interested in knowing how it will display things (dialog, a drawer or whatever), but rather *what* it will display. Better call it `showNotificationDetails` or so.
- Shorten two one line, that bit of code doesn't do enough to deserve so much visual space in the source file :) (you can also drop the `setDialogOpen` from the array if you want, since the React.useState() setters are always immutable and memoized)